### DISCUSSION TITLE
Support for other panels

### DISCUSSION THREAD

**Discussion by dala318** [Blackymas/NSPanel_HA_Blueprint#2134] [Ideas]
Is is possible to install this NSPanel device software on other types and brands like this https://www.aliexpress.com/item/1005006390262736.html
First of all, is the screen compatible? At first glance it don't look like that (NSPanel use nextion while the one above use st7701s)
And secondly, the nspanel_esphome_core.yaml seem to define the two hardware buttons on the NSPanel, while the one above has no such.

**Comment by edwardtfn** [comment]
I have a panel similar to this sitting on my desk since a couple of months ago and couldn't find time to work on it yet.
I can anticipate this will require an entirely new project, as the displays uses very different technologies and this one will require constructing the pages at runtime, which gives way more freedom, but also, way more work.
So, in short, this project cannot be used to non-Nextion based systems, but I wish to find some time to start a new project based on st7701s, of course using the learnings from this one.
